oshi-core/src/main/java/oshi/hardware/CentralProcessor.java

```diff
@@ -165,6 +165,22 @@ public interface CentralProcessor {
      */
     double[] getSystemLoadAverage(int nelem);
 
+    /**
+     * This method waits the provided amount of milliseconds
+     * and then returns the cpu usage for that time period.
+     *
+     * @param delay
+     *          Milliseconds to wait.
+     * @return Value between 0 and 1 (100%) that represents the cpu usage in the provided time period.
+     * @see #getSystemCpuLoadTicks()
+     * @see #getSystemCpuLoadBetweenTicks(long[])
+     */
+    default double getSystemCpuLoad(long delay) {
```

Comment:
For completeness, do you also want to have a `double[] getProcessorCpuLoad(long delay)` method?  (Yeah, you had that before, but this one would return and not do the averaging.)